src/styles/summernote-bs4.scss

```diff
@@ -5,5 +5,4 @@
 .note-toolbar .note-btn {
   background: #fff;
   border-color: #dae0e5;
-  padding: 0.25rem 0.65rem;
```

Comment:
This should not be removed. The default padding of Bootstrap4 is `0.25rem 0.5rem` and it makes buttons too narrow that we expected in Bootstrap3's.

Reply:
I had a look back, and it didn't look to be present before my changes. I can add back.

Reply:
I've amended that change. Sorry for the stuff up.